client/src/components/ProfileSettings/EditProfile/EditProfile.tsx

```diff
@@ -1,7 +1,307 @@
+        marginTop: theme.spacing(8),
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+      },
+      form: {
+        width: '100%', // Fix IE 11 issue.
+        marginTop: theme.spacing(3),
+      },
+      submit: {
+        margin: theme.spacing(3, 0, 2),
+        padding: '0.5rem',
+        width: '10%',
+        color: '#dc004e',
+        border: '1px solid #dc004e',
+      },
+      textbox: {
+        width: '100%',
+      },
+      typography: {
+        fontWeight: 800,
+        fontSize: '12px',
+      },
+      gridcontainer: {
+        padding: theme.spacing(1),
+      },
+      heading: {
+        fontWeight: 550,
+      },
+      phone: {
+        padding: '1px', //TODO add padding or margin later
+      },
+      gender: {
+        fontSize: '10px',
+      },
+    }),
+  );
+  const classes = useStyles();
+  const [phone, setPhone] = useState('');
+  const phoneRegExp =
+    /^((\\+[1-9]{1,4}[ \\-]*)|(\\([0-9]{2,3}\\)[ \\-]*)|([0-9]{2,4})[ \\-]*)*?[0-9]{3,4}?[ \\-]*[0-9]{3,4}?$/;
+  return (
+    <Formik
+      initialValues={{
+        firstName: '',
+        lastName: '',
+        email: '',
+        birthdate: '',
+        address: '',
+        phoneNumber: '',
+        description: '',
+      }}
+      validationSchema={Yup.object({
+        firstName: Yup.string().max(15, 'Must be 15 characters or less'),
+        lastName: Yup.string().max(20, 'Must be 20 characters or less'),
+        email: Yup.string().email('Invalid email address').required('Required'),
+        phoneNumber: Yup.string().matches(phoneRegExp, 'Phone number is not valid'),
+      })}
+      onSubmit={(values, { setSubmitting }) => {
+        values.phoneNumber = phone;
+        console.log(values);
+        setTimeout(() => {
+          alert(JSON.stringify(values, null, 2));
+          axios
+            .post('http://localhost:3001/profiles/create', values)
+            .then((res) => {
+              console.log(res.data);
+            })
+            .catch((error) => {
+              console.log(error);
+            });
+          setSubmitting(false);
+        }, 400);
+      }}
+    >
+      {(formik) => (
+        <form className={classes.form} noValidate onSubmit={formik.handleSubmit}>
+          <Container component="main" maxWidth="sm">
+            <CssBaseline />
+            <div className={classes.paper}>
+              <Box component="div" m={2}>
+                <Typography className={classes.heading} component="h1" variant="h5">
+                  EDIT PROFILE
+                </Typography>
+              </Box>
+
+              <Grid container spacing={3}>
+                <CssBaseline />
+                <Grid container spacing={2} className={classes.gridcontainer}>
+                  <Grid item xs={3}>
+                    <Box component="div" p={1}>
+                      <Typography className={classes.typography} component="h3" variant="h6">
+                        FIRST NAME
```

Comment:
Reserve usage of capital case letters for abbreviations, you can change the text's case using CSS classes here instead. (Ditto throughout). Please see below for a reference on why to avoid capital casing:

Ref: https://medium.com/@mandy.michael/why-you-shouldnt-write-your-content-in-uppercase-instead-use-css-b03ac2c65b99

Reply:
Added "textTransform" css field.